src/en/news/blog/2021/managing-nfs-exports-with-dashboard/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Managing NFS exports with Dashboard"
+date: "2021-12-16"
+author: "Alfonso Martínez"
+image: "images/ganesha.jpg"
+tags:
+  - "dashboard"
+  - "nfs"
+---
+Photo by <a href="https://unsplash.com/@sonika_agarwal?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Sonika Agarwal</a> on <a href="https://unsplash.com/s/photos/ganesha?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+## A bit of History
+
+In the early days, the only way to manage NFS exports was through Ceph-Ansible / OpenStack Manila.
+
+Ceph Dashboard implemented its own NFS export management on nautilus release.
+
+With the advent of the Octopus release, the mgr volume module also implemented the NFS management
+in a similar manner so the user could manage exports via the Orchestrator. Later, NFS management was segregated to the nfs module
+and became the official interface.
+
+So 2 ways of creating exports co-existed: the official implementation (nfs module) and the deprecated ones.
+It was clear that convergence was needed in order to provide a unified and consistent way of managing NFS exports.
+
+## The Convergence is here.
+
+Now in Pacific v16.2.7 release we achieved that convergence so the Dashboard conforms to the nfs module behavior and both
+the GUI and CLI reflect the same information.
+
+Let's do a quick walkthrough through Dashboard NFS section.
+
+When you deploy a cluster, you can go to **Dashboard > NFS > Create Export**. If you don't have any Cluster (NFS service) running, you'll be notified about this:
+
+![NFS Cluster required](images/nfs-clusters-required.png)
+
+So let's go to **Cluster > Services** and create an NFS service:
+
+![Create NFS service](images/create-nfs-service.png)
+
+The NFS service is running:
+
+![NFS service details](images/nfs-service-details.png)
+
+We return to **Dashboard > NFS > Create Export** and create an export with CephFS as the Storage Backend:
```

Comment:
I'd just add level 3 headings for "NFS over CephFS" and "NFS over RGW".

Reply:
Done! I also added `NFS service deployment` header for clarification.